openwisp_controller/config/base/vpn.py

```diff
@@ -657,8 +657,12 @@ class method for ``post_delete`` signal
         # to avoid generating high load during bulk deletes
         instance.vpn._invalidate_peer_cache()
 
-        if instance.cert:
-            instance.cert.delete()
+        try:
+            if instance.cert:
+                instance.cert.delete()
+        except  Exception:
```

Comment:
catching any exception is not a good practice, we need to catch only the specific exception we expect in this case

Reply:
okay sir I am looking forward regarding this matter